avaframe/tests/test_shpConversion.py

```diff
@@ -0,0 +1,84 @@
+"""Tests for module com2AB"""
+import numpy as np
+import os
+import pytest
+
+# Local imports
+import avaframe.in3Utils.ascUtils as ascUtils
+import avaframe.in2Trans.shpConversion as shpConv
+
+
+def test_SHP2Array(capfd):
+    '''Simple test for function SHP2Array'''
+    dirname = os.path.dirname(__file__)
+    shpFileName = os.path.join(dirname, 'data', 'testShpConv', 'testLine.shp')
+    SHPdata = shpConv.SHP2Array(shpFileName, defname=None)
+    Name = SHPdata['Name']
+    Start = SHPdata['Start']
+    Length = SHPdata['Length']
+    Coordx = SHPdata['x']
+    Coordy = SHPdata['y']
+    Coordz = SHPdata['z']
+
+    atol = 1e-10
+    assert Name == ['line1', 'line2']
+    Sol = np.array([0, 3])
+    testRes = np.allclose(Start, Sol, atol=atol)
+    assert (testRes == True)
+    Sol = np.array([3, 3])
+    testRes = np.allclose(Length, Sol, atol=atol)
+    assert (testRes == True)
+    Sol = np.array([9.89017536, 28.11881745, 110.48527428, 13.77220099,
+                    55.29299685, 173.27281924])
+    testRes = np.allclose(Coordx, Sol, atol=atol)
+    assert (testRes == True)
+    Sol = np.array([20.6161196, 51.67232464, 65.17502248, 102.47622527,
+                    84.07879946, 69.05704811])
+    testRes = np.allclose(Coordy, Sol, atol=atol)
+    assert (testRes == True)
+    Sol = np.array([0., 0., 0., 0., 0., 0.])
+    testRes = np.allclose(Coordz, Sol, atol=atol)
+    assert (testRes == True)
```

Comment:
Maybe have a empty line after each test to make it a bit more readable. Or a quick comment what the test checks for...